bedrock/mozorg/templates/mozorg/contribute/contribute-tasks.html

```diff
@@ -0,0 +1,159 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/contribute/contribute-base.html" %}
+
+{% block body_id %}inquiry{% endblock %}
+
+{% block page_title %}{{_('Get Involved: Volunteer Opportunities at Mozilla')}}{% endblock %}
+
+{% block contrib_nav_cta %}{% endblock %}
+{% block contrib_country_form %}{% endblock %}
+
+{% block contrib_title %}
+  <div class="container">
+    <h1 class="page-title">{{ _('Get involved with Mozilla today!') }}</h1>
+    <p class="section-tagline">{{ _('There are a lots of ways to contribute to Mozilla. Pick one below that you think would be right for you.') }}</p>
+  </div>
+{% endblock %}
+
+{% block contrib_content %}
+  {% if variation == 'a1' %}
+    <section class="section inquiry">
+      <div class="container inquiry-choices">
+        <div class="option-group">
+          <h2>{{ _('Only have a little time?') }}</h2>
+          <ul class="option-list">
+            <li class="option">
+              <a data-task-id="1" class="ab-task category category-nightly" href="https://nightly.mozilla.org/" target="_blank"></a>
+              <span class="label-text">{{ _('Help us test by downloading Firefox Nightly') }}</span>
+            </li>
+            <li class="option">
+                <a data-task-id="2" class="ab-task category category-stumbling" href="https://oneanddone.mozilla.org/en-US/tasks/92/" target="_blank"></a>
+                <span class="label-text">{{ _('Download our Stumbler App') }}</span>
```

Comment:
Sorry I didn't notice this before, but these spans should be inside the links so the text is linked as well as the icon. Just moving the <code>&lt;/a&gt;</code> seems to work fine with no other CSS tricks needed, except we might then want to change the link color to match the body text color instead of the regular blue.